lua/nvim-autopairs/conds.lua

```diff
@@ -199,4 +198,16 @@ cond.is_end_line = function()
     end
 end
 
+cond.exclude_filetypes = function(filetypes)
+    return function(opts)
+        log.debug('exclude_filetypes')
+        for _, filetype in pairs(filetypes) do
+            if vim.bo.filetype == filetype then
+                return false
+            end
+        end
+        return true
```

Comment:
you don't need return true here i already say a condition have 3 state to return.

it need to return nil when you are not sure it is correct on the condition

you can check another "not_" condition it doesn't need to return true
Can you change the name to "not_filetypes" to make it similar to other condition

another code is looking good. thank you

Reply:
done